docs/src/Demo.js

```diff
@@ -136,22 +144,19 @@ const HelloWorldview = () => {
         target: [0, 0, 0],
         targetOffset: [0, 0, 0],
         targetOrientation: [0, 0, 0, 1],
-        thetaOffset: 0,
-      }}
-     >
+        thetaOffset: Math.PI / 4,
+        perspective: true,
+      }}>
       <Spheres>
-        {new Array(${numSpheres}).fill().map((_, i) => ({
+        {{
+          points,
+          colors,
           pose: {
+            position: { x:0, y:0, z:0 },
```

Comment:
nit: missing spaces.

Wondering if we should allow `pose` to be left blank and default to ((0,0,0), (0,0,0,1)); thoughts?

Reply:
> Wondering if we should allow pose to be left blank and default to ((0,0,0), (0,0,0,1)); thoughts?

 Yeah that would be nice!